include/boost/gil/algorithm.hpp

```diff
@@ -241,7 +241,7 @@ struct copier_n<iterator_from_2d<IL>,iterator_from_2d<OL>> {
         while (n>0) {
             diff_t l=dst.width()-dst.x_pos();
             diff_t numToCopy=(n<l ? n : l);
-            detail::copy_n(src.x(), numToCopy, dst.x());
+            std::copy_n(src.x(), numToCopy, dst.x());
```

Comment:
> Added line #L244 was not covered by tests

Hmm, this one is interesting. I'd have expected those to be fully covered.

Reply:
I don't see any test cases for `core\algorithm\copy_pixels`. The other specializations of `struct copier_n` are apparently triggered by some unrelated test cases from extension or image processing tests AFAIS.